entity/src/main/java/dev/ikm/tinkar/entity/export/ExportEntitiesToProtobufFile.java

```
@@ -147,8 +147,8 @@ public EntityCountSummary compute() {
             zos.finish();
         } catch (Throwable e) {
             LOG.error("Caught " + e + " while Exporting Entities");
-            AlertStreams.dispatchToRoot(e);
-            throw new RuntimeException(e);
+            // AlertStreams.dispatchToRoot(e);
```

If this fixed the UI problem, we should probably add a separate catch block for InterruptedException to avoid suppressing all exceptions.

---

Yes it did fix it.  ok will add separate catch block for InterruptedException but that might not be the case because the methods before might not be configured to throw that exception, will try it now